sdks/java/io/google-cloud-platform/build.gradle

```diff
@@ -119,9 +120,12 @@ dependencies {
   compile library.java.slf4j_api
   compile library.java.vendored_grpc_1_26_0
   compile library.java.vendored_guava_26_0_jre
+  compile "org.apache.arrow:arrow-memory-core:4.0.0"
```

Comment:
If there's a dependency on the arrow project, do these need to be defined again?

Reply:
It's also needed here since some Arrow objects need to be created[here](https://github.com/apache/beam/blob/56a96a9479d46cab3e7211bbb535ce7b17435f0f/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryStorageArrowReader.java#L54)